micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog 
 
+## [1.9.3]  [21-November-2025]
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)